dev/com.ibm.ws.request.timing.monitor_fat/fat/src/com/ibm/ws/request/timing/monitor/fat/RequestTimingMetricsTest.java

```diff
@@ -96,19 +96,23 @@ public void testBasic() throws Exception {
         String testName = "testBasic";
         Log.info(c, testName, "Entry");
 
-        //Metrics Monitor Bundle has started - using default timeout
-        String logMsg = server.waitForStringInLogUsingMark("CWPMI2003I");
-        Log.info(c, testName, logMsg);
-        Assert.assertNotNull("No CWPMI2003I was found.", logMsg);
-
         //Wait until server has started
-        logMsg = server.waitForStringInLogUsingMark("CWWKF0011I");
-        Log.info(c, testName, logMsg);
-        Assert.assertNotNull("No CWWKF0011I was found.", logMsg);
+        String logMsg = server.waitForStringInLogUsingMark("CWWKF0011I");
```

Comment:
The check for `CWWKF0011I` should be moved to the setUp() as the server is started up once for the entire class.  If another test case ran before this and it set the log mark, this message won't be seen again.